hibike/devices/YogiBear/LED.cpp

```diff
@@ -0,0 +1,60 @@
+#include "LED.h"
```

Comment:
This isn't critical, but at some point we need to install a linter. This file uses tabs, while I think everything else in hibike uses spaces.